### PR TITLE
Fm/check box and buttons

### PR DESCRIPTION
**Pull request by FakharMehdii**
Made components
Added the Strike Through feature
Added the Delete Task feature
Cleaned code

### CODE REVIEW

File: src/Components/Bar/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    function handleCheckboxChange(event) {
+        if(event.target.checked)
+        {
+            setDone(true);
+        }
+        else
+        {
+            setDone(false);
+        }
+
```
Comment (muhammad-qadeer) [inline src/Components/Bar/index.js:17]: your if else condition if contains only 1 line then no need for curly braces

File: src/Components/Bar/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    function handleCheckboxChange(event) {
+        if(event.target.checked)
+        {
+            setDone(true);
+        }
+        else
+        {
+            setDone(false);
+        }
```
Comment (JavedJutt) [inline src/Components/Bar/index.js]: if you have single line in if else statements then no need for curly braces

File: src/App.js [hunk 3]
```diff
@@ -1,88 +1,47 @@
...
+    if (event.key === 'Enter' && inputValue!=="") {
+      console.log("Enter pressed");
+      setTasks((prevValues) => [...prevValues, inputValue.trim()]);
+      setInputValue('');
+    }
+  }
 
   return (
-    <ReturnApp inputValue={inputValue} handleChange={handleChange} handleKeyPress={handleKeyPress} handleDelete={handleDelete} handleEdit= {handleEdit} />
+    <RenderApp handleChange={handleChange} setTasks={setTasks}  tasks={tasks} inputValue={inputValue} handleKeyPress={handleKeyPress} handleDelete={handleDelete}  />
```
Comment (JavedJutt) [inline src/App.js]: make this component meaningful name

File: src/App.css [hunk 4]
```diff
@@ -7,6 +7,11 @@
   pointer-events: none;
 }
 
+.strike {
+  text-decoration: line-through;
+}
+
+
```
Comment (JavedJutt) [inline src/App.css]: remove extra white spaces

File: src/Components/Bar/index.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+    if(edit)
+    {
+        
+        return(
+            <div key={index} className='bar'>
+                
+                <div>
+        
+                <input   className="cbox" type='text' value={tasks[index]} onChange={handleChange} onKeyDown={(event) => handleKeyPress(event, index)} autoFocus />
```
Comment (JavedJutt) [inline src/Components/Bar/index.js]: use meaningful classes name

File: src/Components/InputField/index.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+
+
+
```
Comment (JavedJutt) [inline src/Components/InputField/index.js]: again dummy lie breaks

File: src/Components/TodoArea/index.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { InputField } from "../InputField"
+import { Bar } from "../Bar"
+
+export function TodoArea ({inputValue, handleChange, handleKeyPress, setTasks, tasks, handleDelete})
+{
```
Comment (JavedJutt) [inline src/Components/TodoArea/index.js]: TodoArea doesn't make any sense, use proper names

**Comment by JavedJutt** [comment]
left comments, resolve and complete these so that we can move towards redux

**Pull request merged**